cli/cmd/root.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/rilldata/rill/cli/cmd/runtime"
 	"github.com/rilldata/rill/cli/cmd/source"
 	"github.com/rilldata/rill/cli/cmd/start"
+	"github.com/rilldata/rill/cli/cmd/superuser/autoinvite"
```

Comment:
The idea of a superuser namespace is nice, let's make it more explicit. Suggested refactors:

1. Rename `.../cmd/superuser` to `.../cmd/sudo`
2. Mount a `SudoCmd` on `rill sudo`, and make it `Hidden: true`
3. Then put the auto invite commands under that, like `rill sudo autoinvite add`